app/src/main/java/com/amaze/filemanager/utils/NetworkUtil.kt

```diff
@@ -115,7 +116,8 @@ object NetworkUtil {
                 netinterface.inetAddresses.iterator().forEach { address ->
                     // this is the condition that sometimes gives problems
                     if (!address.isLoopbackAddress &&
-                        !address.isLinkLocalAddress
+                        !address.isLinkLocalAddress &&
+                        address is Inet4Address
```

Comment:
this is just a type check, right? does this "force" Ipv4 address?  🤔 I'll send the APK to the reporter & ask them to check.